docs/API.md

```diff
@@ -165,12 +165,53 @@ An `<IndexLink>` is like a [`<Link>`](#link), except it is only active when the
 ### `<RouterContext>`
 A `<RouterContext>` renders the component tree for a given router state. Its used by `<Router>` but also useful for server rendering and integrating in brownfield development.
 
-It also provides a `router` object on `context`.
+It also provides a `router` object on [context](https://facebook.github.io/react/docs/context.html).
 
 #### `context.router`
 
 Contains data and methods relevant to routing. Most useful for imperatively transitioning around the application.
 
+To use it, you must signal to React that you need it by declaring your use of it in your component:
+
+```js
+var MyComponent = React.createClass({
+  contextTypes: {
+    router: React.PropTypes.object.isRequired
```

Comment:
@timdorr wdyt of renaming this to `routerShape` so users can just do

```js
import { routerShape } from 'react-router'
```

without any issues with collisions, or having to do e.g.

```js
import { * as Router } from 'react-router'
```

?